storage/maria/ma_check.c

```diff
@@ -4250,6 +4250,8 @@ int maria_repair_parallel(HA_CHECK *param, register MARIA_HA *info,
     }
   */
   DBUG_PRINT("info", ("is quick repair: %d", (int) rep_quick));
+  if (!rep_quick)
+    my_b_clear(&new_data_cache);
```

Comment:
Taking into account simplicity of ```my_b_clear()``` and ```my_b_inited()``` I'd rather do ```my_b_clear()``` unconditionally. It saves 2 conditional jumps, which performance wise are worse than single unconditional store.

This is just to voice my opinion, current implementation is generally fine too. No action needed.